doc/contributing.rst

```diff
@@ -0,0 +1,171 @@
+
+* Writing `meaningful commit messages <https://www.conventionalcommits.org/en/v1.0.0/>`_ can be very helpful for you and people who review your code to better understand the code changes.
+
+
+=========================
+Our branching strategy
+=========================
+
+While you can use any type of branching strategy and naming as you work in your personal fork, we have three branches in the tobac repository: 
+
+* :code:`RC_*`
+* :code:`dev_*`
+* :code:`hotfix`
+
+:code:`RC_*` is the release candidate of the next tobac version. The asterisk stands here for the specific tobac version: RC_vx.x.x (e.g. RC_v1.5.0). Pull requests to this branch need two reviews to be accepted before it can be merged into main. 
+
+:code:`dev_*` is the development branch where we experiment with new features. This branch is perfectly suited to collaboratively work on a feature together with other **tobac** developers (see :doc:`mentoring`). In general, this branch is used for long-term, comprehensive code changes that might not be covered by a single pull request and where it might not be conceivable in which future **tobac** version to include the changes. There are no branch protection rules for this branch, which means that collaborators of our GitHub organization can directly push changes to this branch. Note that **dev_** can never directly merged into main, it has be merged into the release candidate branch :code:`RC_*` first! There can be more than one `dev_*` branch, therefore it we recommand to describe the feature to work on in the respective branch (e.g. :code:`dev_xarray_transition`). 
+
+:code:`hotfix` is the branch we use for hotfixes, i.e. bug fixes that need to be released as fast as possible because it influences people's code. This branch needs only one review before it can directly merged into :code:`main`.
+
+In brief: **Unless you are collaboratively working on a comprehensive feature or on a hotfix, the branch to submit your pull request to is the next release candidate RC_v.x.x.x**
+
+
+=========================
+GitHub workflow
+=========================
+
+We use several [GitHub actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) to 
+assure continuous integration and to enable an efficient code development and release process. Our workflow 
+configuration can be found in 
+[.github/workflows](https://github.com/tobac-project/tobac/tree/main/.github/workflows) and encompass
+
+* check that code is formatted using the latest stable version of black 
+* linting of the latest code changes that checks the code quality and results in a score compared to the most recent 
+released version 
+* check of the zenodo JSON file that ensures that the citation is correct 
+* check that all unit tests pass (including testing on multiple operating testings) and report test coverage 
+* check that the example jupyter notebooks run without problems 
+* check that the readthedocs page is properly built 
```

Comment:
Not sure this is true

Reply:
I mean the action on RTD rendering

Reply:
Good point, I think we are actually not testing this. And i only now saw your detailed description on the sphinx-based rendering and how you can check the RTC page for pull requests etc. So I just removed this point here and think your descriptions suffice.